student/models.py

```diff
@@ -24,6 +24,8 @@ class Student(models.Model):
     phone_number = models.CharField(max_length=15, blank=True)
     educational_institution = models.ForeignKey(EducationalInstitution, on_delete=models.PROTECT, default=None,
                                                 related_name="students")
+    major = models.CharField(max_length=255)
```

Comment:
Wouldn't you want to make major a selectable value as opposed to free text?

Reply:
Umm, yea I guess I could add that, on it.

Reply:
Done, I added a few majors to choose from we can add more in the future if we feel like it, for now, I think it'll serve its purpose